orangecontrib/text/tag/pos.py

```diff
@@ -52,3 +86,52 @@ class MaxEntTagger(POSTagger):
     def __init__(self):
         tagger = nltk.data.load('taggers/maxent_treebank_pos_tagger/english.pickle')
         super().__init__(tagger)
+
+
+def find_model(language: str) -> str:
+    return SPACY_MODELS[language]["package"]
+
+
+class SpacyModels:
+    installed_models_info = info()
+
+    def __init__(self):
+        self.installed_models = self.installed_models_info['pipelines']
+
+    def __getitem__(self, language: str) -> str:
+        model = find_model(language)
+        if model not in self.installed_models:
+            download(model)
```

Comment:
`self.installed_models` should be updated at this point. If not the package keeps getting downloaded.